package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "biocaml",
+  "version": "0.11.3",
```

Comment:
We normally put "dev" for the version, except in the commit that represents the actual release.

Reply:
Done: https://github.com/biocaml/biocaml/compare/b5f164db11cbb27685323c1b6c35faf4b7002b9c..f3d5feef07881fc97c4cd92793116ce63e88addd#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R3